app/classifier/tasks/drawing/summary.cjsx

```diff
@@ -1,4 +1,6 @@
 React = require 'react'
+strip = require 'strip-markdown'
+remark = (require 'remark').use(strip)
```

Comment:
I looked into this. Looks like `strip-markdown` is a specific plugin written for `remark`. We use https://github.com/zooniverse/markdownz which is built upon https://github.com/markdown-it/markdown-it. While `markdown-it` does have [plugins](https://www.npmjs.com/browse/keyword/markdown-it-plugin), there doesn't seem to be one equivalent to `strip-markdown`. So there isn't an easy way to avoid using `remark` without writing our own version of `strip-markdown` to work on `markdown-it`. I think we will have to put up with inconvenience of loading `remark` for now. It's only for this one single `.cjsx` file, not the whole classifier....